creator-node/src/config.js

```diff
@@ -378,6 +378,12 @@ const config = convict({
     format: 'nat',
     env: 'rehydrateMaxConcurrency',
     default: 10
+  },
+  triggerSyncOnWrite: {
```

Comment:
why not change this config to be more clearly dev like 'snapbackDevModeEnabled' and then use that to either schedule cron or a repeated job (like is currently enabled)?